kbkdf/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "kbkdf"
+version = "0.1.0"
+edition = "2018"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/RustCrypto/KDFs/"
+repository = "https://github.com/RustCrypto/KDFs/"
+description = "Key Derivation Using Pseudorandom Function (KBKDF)"
+keywords = ["crypto", "KBKDF", "KDF"]
+categories = ["cryptography", "no-std"]
+readme = "README.md"
+rust-version = "1.65"
+
+[dependencies]
+digest = { version = "0.11.0-pre.9", default-features = false, features = ["mac"] }
+
+# divrem can be dropped with MSRV >= 1.73
+divrem = "1.0.0"
+
+[dev-dependencies]
+hex-literal = "0.2.2"
```

Comment:
The latest version of `hex-literal` is v0.4. If there's anything MSRV-related keeping you from using the latest versions, please bump MSRV.

Reply:
done